README.md

```diff
@@ -1,87 +1,5 @@
-
-## Pull request format
-If you are submitting a pull request for **mainnet**, you must write a summary in the notes of the PR that describes how you are ensuring **data persistence** of your (1) multiaddress, (2) Ceramic State Store, and (3) IPFS Repo, and includes the static IP address of the machine your Ceramic daemon runs on so it can connect to the 3Box Labs hosted Ceramic Anchor Service (CAS).
-
-```md
-### Team:
-<!--Team name or your github handle if you are a team of one-->
-### Use case:
-<!--A few words about what how your node will be used so we can make recommendations for your setup-->
-
-### Overview:
-<!--How are you running your nodes? What cloud infrastructure? Are you running IPFS out-of-process?-->
-
-*Multiaddress persistence:*
-<!--What are you doing to ensure your multiaddress won't change?-->
-
-*Ceramic State Store persistence:*
-<!--What are you doing to ensure your Ceramic data doesn't get lost?-->
-
-*IPFS Repo persistence:*
-<!--What are you doing to ensure your IPFS data doesn't get lost?-->
-
-*Static IP:*
-<!--Static IP address of the machine your Ceramic daemon runs on.-->
-```
-
-> **Data persistence is a critical step.** Remember that there are no guarantees that other nodes in the network are keeping copies of your streams so you must pin all streams that you care about and back up their data.
->
-> If your nodes restart for any reason and you *do not* have data persistence in place, the following will occur: (1) Your IPFS `config` file will be regenerated with a new multiaddress and you will lose your connection to the rest of the network. (2) The state data for streams pinned on your node will get deleted and likely become unrecoverable. (3) The IPFS data for streams your node has loaded will get deleted and likely become unrecoverable, corrupting your stream state.
->
-> Instructions on configuring your node for proper data persistence can be found in the Ceramic docs on the [Hosting a node](https://developers.ceramic.network/run/nodes/nodes/) page.
-
-Please add one of the maintainers below as a reviewer for your PR. They will do their best to merge PRs as soon as they come in. If you need more immediate assistance, reach out on [Discord](https://chat.ceramic.network) and include the URL of your PR.
-
-## Why peerlist?
-
-Peerlist is a *temporary* solution for peer discovery in Ceramic that will be used up until DAG-JOSE becomes available in go-ipfs. Peerlist is needed because the current JavaScript implementation of the DHT in `libp2p` (a p2p networking library used in Ceramic) is not yet sufficient to deliver optimal performance for a live decentralized network.
-
-## Maintainers
-
-- Val ([@v-stickykeys](https://github.com/v-stickykeys))
-
-## License
-
-Ceramic is fully open source and dual-licensed under MIT and Apache 2.
+# ⛔️ This repo has been superseded by the [Ceramic Anchor Allowlist](https://github.com/3box/ceramic-anchor-allowlist)
```

Comment:
I don't think this notice is accurate yet. We aren't really deprecating this repo and the new one doesn't replace this one entirely in terms of how it functions.

Would prefer to just have this be a short description of the peerlist and a note like what you have below:

"Note, if you are looking to join Ceramic mainnet and would like to be added to the 3Box Labs Anchor service, please...."